.drone.yml

```diff
@@ -0,0 +1,21 @@
+kind: pipeline
```

Comment:
I think we can remove `.drone.yml` as we'll using Harness CI for testing and publishing plugins.

Reply:
Done